scripts/partition_manager_output.py

```diff
@@ -24,6 +24,15 @@ def get_header_guard_end(filename):
 DEST_HEADER = 1
 DEST_KCONFIG = 2
 
+# If nl is something like /soc/peripheral@50000000/qspi@2b000/mx25r6435f@0, it will be
+# converted to C define for node, in this example
+# DT_N_S_soc_S_peripheral_50000000_S_qspi_2b000_S_mx25r6435f_0.
```

Comment:
we should not rely on how Devicetree structures definitions internally, this is an implementation detail that could change at any time.

Reply:
Yes, this is unfortunate, but this happens when cmake passes result of `dt_chosen` to script; what you get is node path, as it even evaluates `&label` to path.